src/plugins/intel_gpu/tests/fusions/convolution_fusion_test.cpp

```diff
@@ -1817,7 +1817,7 @@ TEST_P(conv_int8_prelu_eltwise, basic_slope_2) {
         reorder("reorder_bfyx", input_info("eltwise"), p.default_format, data_types::f32)
     );
 
-    tolerance = default_tolerance(p.default_type);
+    tolerance = 1.5;
```

Comment:
Let's change implementation in `default_tolerance`

Reply:
Done.